salt/edx/xsiftx.sls

```diff
@@ -28,11 +28,11 @@
 
 install_xsiftx:
   pip.installed:
-    - name: git+https://{{ repo }}@{{ version }}#egg=xsiftx
+    - name: git+{{ repo }}@{{ version }}#egg=xsiftx
     - exists_action: w
 
-{% if extra_sifter_repo is not None %}
-install_extra_sifters
+{% if extra_sifter_repo is not none %}
```

Comment:
Thanks, I'll change this. I tend habitually be more explicit when using Jinja2 due to some weird Ansible behavior with vars.